src/utils/time.ts

```diff
@@ -190,6 +190,20 @@ export function localizeTimeStringToParts(value: string, locale = "en"): Localiz
   return null;
 }
 
+export function getformatToParts(value: string, locale = "en"): Intl.DateTimeFormatPart[] {
+  if (!isValidTime(value)) {
+    return null;
+  }
+  const { hour, minute, second = "0" } = parseTimeString(value);
+  const dateFromTimeString = new Date(Date.UTC(0, 0, 0, parseInt(hour), parseInt(minute), parseInt(second)));
```

Comment:
Could you use a constant instead of creating a new date every time? We're interested in the part order, so the actual time isn't crucial.

Reply:
we cannot use a constant here as few locales have diff meridiem symbols based on the time of the day.

Reply:
That's good to know. Thanks for the info.